src/NativeScript/GlobalObject.moduleLoader.mm

```diff
@@ -390,6 +393,15 @@
                                                   args.append(moduleKey);
                                                   JSValue entry = JSC::call(execState, function, callType, callData, moduleLoader, args);
                                                   record = jsCast<JSModuleRecord*>(entry.get(execState, Identifier::fromString(execState, "module")));
+                          
+                                                  if (frame.check()) {
```

Comment:
There is one-liner `TNSPERF()` that does `tns::instrumentation::Frame frame(__func__)` that will print the method name if the method takes too much time, this better be placed the the start of a method and relies on the Frame C++ destructor to mark the end of the method.

The if-ed stuff is done to minimise runtime costs when running without profiling. Most of the if-ed logic in the places where we are mixing logic is stuff that will extract string and combine them, that is cost that we won't pay if the profiling is not enabled or if the frame was actually fast (<16ms.).

I have no idea what you may need one day. I'd be glad for further contribution on this.

`checkForMyCustomEvent1WithFrame::withCustomParam:` and the rest will make the profile know about random stuff in the runtime, I don't think the Frame should be coupled with anything else but timing and logging.